dwave/cloud/utils.py

```diff
@@ -867,11 +873,52 @@ def get_contrib_packages():
 def get_platform_tags():
     """Return a list of platform tags generated from registered entry points."""
 
-    fs = [ep.load() for ep in iter_entry_points('dwave.common.platform.tags')]
+    fs = [ep.load() for ep in entry_points(group='dwave.common.platform.tags')]
     tags = list(filter(None, [f() for f in fs]))
     return tags
 
 
+class VersionNotFoundError(Exception):
+    """Package version requirement not satisfied."""
```

Comment:
Did you mean do expand this class definition?

Reply:
No, I think this is more than enough. Have in mind this is an internal, undocumented exception raised by an internal, undocumented function (`get_distribution`).